lms/envs/production.py

```diff
@@ -1137,6 +1137,31 @@
     COLARAZ_PREVIEW_DOMAIN_KEY
 )
 
+# Custom urls in navbar
+COLARAZ_RECRUITMENT_URL = ENV_TOKENS.get(
+    'COLARAZ_RECRUITMENT_URL'
+)
```

Comment:
nit: we can add fallback variables in `common.py` with a default value and use them here:
```suggestion
    COLARAZ_RECRUITMENT_URL = ENV_TOKENS.get('COLARAZ_RECRUITMENT_URL', COLARAZ_RECRUITMENT_URL)
```

Reply:
I have thought about that but we don't have any consistent fallback URL at the moment.